spec/unit/webrtc/callFeed.spec.ts

```diff
@@ -85,5 +85,60 @@ describe("CallFeed", () => {
                 expect(feed.isVideoMuted()).toBeTruthy();
             });
         });
+
+        describe("voice activity detection", () => {
```

Comment:
The test names could be more descriptive here, also, would it be possible to test the cooldown?

Reply:
Done :heavy_check_mark: